lms/templates/instructor/instructor_dashboard_2/cohorts.underscore

```diff
@@ -40,11 +40,13 @@
         <div class="cohort-management-supplemental">
             <p class="">
                 <i class="icon fa fa-info-circle" aria-hidden="true"></i>
-                <%= interpolate(
-                gettext('To review student cohort assignments or see the results of uploading a CSV file, download course profile information or cohort results on %(link_start)s the Data Download page. %(link_end)s'),
-                    {link_start: '<a href="" class="link-cross-reference" data-section="data_download">', link_end: '</a>'},
-                    true
-                ) %>
+                <%= HtmlUtils.interpolateHtml(
```

Comment:
Nit: the indentation is a little confusing. I'd go with:

    <%=
    HtmlUtils.interpolateHtml(
        gettext('...),
        ...
    );
    %>